packages/emptystate/copySvgs.js

```diff
@@ -0,0 +1,14 @@
+const ncp = require('ncp').ncp
```

Comment:
why use `npc` instead of just using the `cp -r` command from the terminal. is this better because it works on windows - or are there other reasons

Reply:
Yep, in the past Jake reminded me as this is an opensource project we should try to be good citizens. Otherwise I would of gone with cp -r.